hrm-service/src/contact/sql/contact-update-sql.ts

```diff
@@ -37,3 +37,13 @@ RETURNING *
 )
 ${selectSingleContactByIdSql('updated')}
 `;
+
+export const UPDATE_CASEID_BY_ID = `WITH updated AS (
+UPDATE "Contacts" 
+SET 
+  "caseId" = $<caseId>
+WHERE "accountSid" = $<accountSid> AND "id"=$<contactId>
+RETURNING *
+)
+${selectSingleContactByIdSql('updated')}
```

Comment:
Why this `select` after the `update`? Does `returning *` misses something after the query?

Reply:
`RETURNING` can only return the record that was inserted, but nothing joined to it. Our API requires returning the CSAM Reports as well, so I use a CTE to join the additional lookups to the updated record